mirte_telemetrix/scripts/ROS_telemetrix_aio_api.py

```diff
@@ -622,11 +622,13 @@ async def start(self):
             SetOLEDImage,
             self.set_oled_image_service,
         )
-
         for ev in self.init_awaits:
             await ev
         for cmd in self.write_commands:
+            # // TODO: arduino will just stop forwarding i2c write messages after a single failed message. No feedback from it yet.
```

Comment:
Could we create a new issue for this, just to keep in on our radar.

Reply:
Done!